test/nodetool/test_describering.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+from rest_api_mock import expected_request
+
+def test_describering(nodetool):
+    schema_version = "83541f18-c1bc-11ee-b55e-4d563ca8da4b"
+    ring = [
+            {
+                "start_token": "-9153143965931359657",
+                "end_token": "-9105705820509211664",
+                "endpoints": [
+                    "127.0.0.1",
+                    "127.0.0.2"
+                ],
+                "rpc_endpoints": [
+                    "127.0.0.1",
+                    "127.0.0.2"
+                ],
+                "endpoint_details": [
+                    {
+                        "host": "127.0.0.1",
+                        "datacenter": "datacenter1",
+                        "rack": "rack1"
+                    },
+                    {
+                        "host": "127.0.0.2",
+                        "datacenter": "datacenter2",
+                        "rack": "rack2"
+                    }
+                ]
+            },
+            {
+                "start_token": "9213626581013704850",
+                "end_token": "-9153143965931359657",
+                "endpoints": [
+                    "127.0.0.1"
+                ],
+                "rpc_endpoints": [
+                    "127.0.0.1"
+                ],
+                "endpoint_details": [
+                    {
+                        "host": "127.0.0.1",
+                        "datacenter": "datacenter1",
+                        "rack": "rack1"
+                    }
+                ]
+            }
+    ]
+    res = nodetool("describering", "ks", expected_requests=[
+        expected_request("GET", "/storage_service/schema_version", response=schema_version),
```

Comment:
Why do you expect the schema version cannot change?

Reply:
I don't follow. This is just to check that the version sent by the server is printed.

Reply:
Me confused. I thought schema_version was the version you expect from the server, but it's both the version you expect from the server and the response you preload into the mock server.

Reply:
Yes.